CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
 # Changelog
 
 ## [0.12.0](https://github.com/brave/browser-laptop/releases/v0.12.0dev)
-- (todo)
+ - Added shortcut to view page source ([#3685](https://github.com/brave/browser-laptop/issues/3685))
+ - Added extra separator under bookmark menu ([#3666](https://github.com/brave/browser-laptop/issues/3666))
+ - Added an option to temporarily disable mixed content protection enhancement security ([#3443](https://github.com/brave/browser-laptop/issues/3443))
+ - Changed friendlier add funds panel for users without a bitcoin: handler ([#3623](https://github.com/brave/browser-laptop/issues/3623))
+ - Changed search settings panel: search bar options ([#2124](https://github.com/brave/browser-laptop/issues/2124))
+ - Fixed many things for Brave Payments ([#3697](https://github.com/brave/browser-laptop/issues/3697))([#3625](https://github.com/brave/browser-laptop/issues/3625))([#3613](https://github.com/brave/browser-laptop/issues/3613))([#3638](https://github.com/brave/browser-laptop/issues/3638))
+ - Fixed spelling for Ukrainian interface ([#3695](https://github.com/brave/browser-laptop/issues/3695))
+ - Fixed wrong suggestions appearing in the autocomplete list under URL bar ([#3643](https://github.com/brave/browser-laptop/issues/3643))
+ - Fixed potential race condition on session state save ([#3543](https://github.com/brave/browser-laptop/issues/3543))
+ - Fixed passwords.properties not being loaded at app level ([#3671](https://github.com/brave/browser-laptop/issues/3671))
```

Comment:
People don't know what passwords.properties is, so say fixed localization for password strings as that title